frontend/src/components/eventsModal.js

```diff
@@ -0,0 +1,94 @@
+import { Component } from 'react'
+import Router from 'next/router'
+import {
+  Alert,
+  Button,
+  Form,
+  FormGroup,
+  Label,
+  Input,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter
+} from 'reactstrap'
+import { createEvent } from '../utils/api'
+
+class EventsModal extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      alert: ''
+    }
+  }
+
+  addEvent = async () => {
+    const event = {
+      name: this.state.name ? this.state.name : '',
```

Comment:
replace `this.state.name ? this.state.name : ''` with `this.state.name`, server should catch this error and throw a response